### DISCUSSION TITLE
无法构建ClassIsland项目且提示“缺少某些程序集引用。进行生成以还原 NuGet 缓存可能会解决此问题。”

### DISCUSSION THREAD

**Discussion by AWDNDWindowsXP** [ClassIsland/ClassIsland#476] [Q&A] [answered]
严重性	代码	说明	项目	文件	行	禁止显示状态	详细信息
错误(活动)	MSB4057	该项目中不存在目标“GitVersion”。	ClassIsland, ClassIsland.Core, ClassIsland.DocsGenerator, ClassIsland.PluginSdk, ClassIsland.Shared (net472), ClassIsland.Shared (net8.0)	D:\ClassIsland-master\Global.props	13		

严重性	代码	说明	项目	文件	行	禁止显示状态	详细信息
警告(活动)	MC3074	XML 命名空间“http://classisland.tech/schemas/xaml/core”中不存在标记“SettingsPageBase”。 行 1 位置 22.	ClassIsland.ExamplePlugin	D:\ClassIsland-master\ClassIsland.ExamplePlugin\Views\SettingsPages\HelloSettingsPage.xaml	1		

![image](https://github.com/user-attachments/assets/8b0a604c-09d8-4159-a72a-0f94ec57e92b)

**Comment by HelloWRC** [comment, marked as answer]
https://docs.classisland.tech/dev/get-started/devlopment.html#%E7%BC%96%E8%AF%91%E4%B8%8E%E8%BF%90%E8%A1%8C

首次编译需要根据文档的指示进行额外的步骤。

**Reply by AWDNDWindowsXP** [reply]
生成还是失败了

严重性	代码	说明	项目	文件	行	禁止显示状态	详细信息
错误(活动)	MSB4044	未给任务“GetAssemblyVersion”的必需参数“NuGetVersion”赋值。	ClassIsland, ClassIsland.Core, ClassIsland.Shared (net472), ClassIsland.Shared (net8.0), ClassIsland.Shared.IPC (net472), ClassIsland.Shared.IPC (net8.0-windows)	C:\Program Files\dotnet\sdk\8.0.403\Sdks\Microsoft.NET.Sdk\targets\Microsoft.NET.GenerateAssemblyInfo.targets	226		
错误(活动)	MC3066	类型引用无法找到名为“Subject”的公共类型。 行 42 位置 71.	ClassIsland.Core	D:\ClassIsland-master\ClassIsland.Core\Controls\LessonsControls\LessonControlExpanded.xaml	42		
错误(活动)	MC3066	类型引用无法找到名为“Subject”的公共类型。 行 25 位置 67.	ClassIsland.Core	D:\ClassIsland-master\ClassIsland.Core\Controls\LessonsControls\LessonControlMinimized.xaml	25		
错误(活动)	MC3066	类型引用无法找到名为“TimeLayoutItem”的公共类型。 行 37 位置 78.	ClassIsland.Core	D:\ClassIsland-master\ClassIsland.Core\Controls\LessonsControls\LessonsListBox.xaml	37		
错误(活动)	MC3074	XML 命名空间“http://classisland.tech/schemas/xaml/core”中不存在标记“AppBase”。 行 1 位置 13.	ClassIsland	D:\ClassIsland-master\ClassIsland\App.xaml	1		
错误(活动)	MC6020	标记“AttachedSettingsControlBase”上的 x:TypeArguments='attachedSettings:AfterSchoolNotificationAttachedSettings' 无效。要么“AttachedSettingsControlBase”不是泛型类型，要么特性中 Type 参数的数目错误。请移除 x:TypeArguments 特性，因为只在通用类型上允许该特性，或将其值修复为匹配泛型类型“AttachedSettingsControlBase”的 arity。 行 13 位置 74.	ClassIsland	D:\ClassIsland-master\ClassIsland\Controls\AttachedSettingsControls\AfterSchoolNotificationAttachedSettingsControl.xaml	13		
错误(活动)	MC6020	标记“AttachedSettingsControlBase”上的 x:TypeArguments='attachedSettings:ClassNotificationAttachedSettings' 无效。要么“AttachedSettingsControlBase”不是泛型类型，要么特性中 Type 参数的数目错误。请移除 x:TypeArguments 特性，因为只在通用类型上允许该特性，或将其值修复为匹配泛型类型“AttachedSettingsControlBase”的 arity。 行 16 位置 74.	ClassIsland	D:\ClassIsland-master\ClassIsland\Controls\AttachedSettingsControls\ClassNotificationAttachedSettingsControl.xaml	16		
错误(活动)	MC6020	标记“AttachedSettingsControlBase”上的 x:TypeArguments='attachedSettings:LessonControlAttachedSettings' 无效。要么“AttachedSettingsControlBase”不是泛型类型，要么特性中 Type 参数的数目错误。请移除 x:TypeArguments 特性，因为只在通用类型上允许该特性，或将其值修复为匹配泛型类型“AttachedSettingsControlBase”的 arity。 行 17 位置 74.	ClassIsland	D:\ClassIsland-master\ClassIsland\Controls\AttachedSettingsControls\LessonControlAttachedSettingsControl.xaml	17		
错误(活动)	MC6020	标记“AttachedSettingsControlBase”上的 x:TypeArguments='attachedSettings:WeatherNotificationAttachedSettings' 无效。要么“AttachedSettingsControlBase”不是泛型类型，要么特性中 Type 参数的数目错误。请移除 x:TypeArguments 特性，因为只在通用类型上允许该特性，或将其值修复为匹配泛型类型“AttachedSettingsControlBase”的 arity。 行 15 位置 74.	ClassIsland	D:\ClassIsland-master\ClassIsland\Controls\AttachedSettingsControls\WeatherNotificationAttachedSettingsControl.xaml	15		
错误(活动)	MC3074	XML 命名空间“http://classisland.tech/schemas/xaml/core”中不存在标记“StringToRadioButtonSelectionConverter”。 行 80 位置 18.	ClassIsland	D:\ClassIsland-master\ClassIsland\Controls\ClassPlanGroupItemControl.xaml	80		
错误(活动)	MC6020	标记“ComponentBase”上的 x:TypeArguments='componentSettings:ClockComponentSettings' 无效。要么“ComponentBase”不是泛型类型，要么特性中 Type 参数的数目错误。请移除 x:TypeArguments 特性，因为只在通用类型上允许该特性，或将其值修复为匹配泛型类型“ComponentBase”的 arity。 行 12 位置 66.	ClassIsland	D:\ClassIsland-master\ClassIsland\Controls\Components\ClockComponent.xaml	12		
错误(活动)	MC6020	标记“ComponentBase”上的 x:TypeArguments='componentSettings:ClockComponentSettings' 无效。要么“ComponentBase”不是泛型类型，要么特性中 Type 参数的数目错误。请移除 x:TypeArguments 特性，因为只在通用类型上允许该特性，或将其值修复为匹配泛型类型“ComponentBase”的 arity。 行 12 位置 67.	ClassIsland	D:\ClassIsland-master\ClassIsland\Controls\Components\ClockComponentSettingsControl.xaml	12		
错误(活动)	MC6020	标记“ComponentBase”上的 x:TypeArguments='componentSettings:CountDownComponentSettings' 无效。要么“ComponentBase”不是泛型类型，要么特性中 Type 参数的数目错误。请移除 x:TypeArguments 特性，因为只在通用类型上允许该特性，或将其值修复为匹配泛型类型“ComponentBase”的 arity。 行 16 位置 83.	ClassIsland	D:\ClassIsland-master\ClassIsland\Controls\Components\CountDownComponent.xaml	16		
错误(活动)	MC6020	标记“ComponentBase”上的 x:TypeArguments='componentSettings:CountDownComponentSettings' 无效。要么“ComponentBase”不是泛型类型，要么特性中 Type 参数的数目错误。请移除 x:TypeArguments 特性，因为只在通用类型上允许该特性，或将其值修复为匹配泛型类型“ComponentBase”的 arity。 行 17 位置 67.	ClassIsland	D:\ClassIsland-master\ClassIsland\Controls\Components\CountDownComponentSettingsControl.xaml	17		
错误(活动)	MC3074	XML 命名空间“clr-namespace:ClassIsland.Core.Abstractions.Controls;assembly=ClassIsland.Core”中不存在标记“ComponentBase”。 行 1 位置 25.	ClassIsland	D:\ClassIsland-master\ClassIsland\Controls\Components\DateComponent.xaml	1		
错误(活动)	MC6020	标记“ComponentBase”上的 x:TypeArguments='componentSettings:LessonControlSettings' 无效。要么“ComponentBase”不是泛型类型，要么特性中 Type 参数的数目错误。请移除 x:TypeArguments 特性，因为只在通用类型上允许该特性，或将其值修复为匹配泛型类型“ComponentBase”的 arity。 行 12 位置 55.	ClassIsland	D:\ClassIsland-master\ClassIsland\Controls\Components\ScheduleComponent.xaml	12		
错误(活动)	MC6020	标记“ComponentBase”上的 x:TypeArguments='componentSettings:LessonControlSettings' 无效。要么“ComponentBase”不是泛型类型，要么特性中 Type 参数的数目错误。请移除 x:TypeArguments 特性，因为只在通用类型上允许该特性，或将其值修复为匹配泛型类型“ComponentBase”的 arity。 行 14 位置 60.	ClassIsland	D:\ClassIsland-master\ClassIsland\Controls\Components\ScheduleComponentSettingsControl.xaml	14		
错误(活动)	MC3074	XML 命名空间“http://classisland.tech/schemas/xaml/core”中不存在标记“ComponentBase”。 行 1 位置 19.	ClassIsland	D:\ClassIsland-master\ClassIsland\Controls\Components\SeparatorComponent.xaml	1		
错误(活动)	MC6020	标记“ComponentBase”上的 x:TypeArguments='componentSettings:TextComponentSettings' 无效。要么“ComponentBase”不是泛型类型，要么特性中 Type 参数的数目错误。请移除 x:TypeArguments 特性，因为只在通用类型上允许该特性，或将其值修复为匹配泛型类型“ComponentBase”的 arity。 行 12 位置 66.	ClassIsland	D:\ClassIsland-master\ClassIsland\Controls\Components\TextComponent.xaml	12		
错误(活动)	MC6020	标记“ComponentBase”上的 x:TypeArguments='componentSettings:TextComponentSettings' 无效。要么“ComponentBase”不是泛型类型，要么特性中 Type 参数的数目错误。请移除 x:TypeArguments 特性，因为只在通用类型上允许该特性，或将其值修复为匹配泛型类型“ComponentBase”的 arity。 行 11 位置 66.	ClassIsland	D:\ClassIsland-master\ClassIsland\Controls\Components\TextComponentSettingsControl.xaml	11		
错误(活动)	MC6020	标记“ComponentBase”上的 x:TypeArguments='componentSettings:WeatherComponentSettings' 无效。要么“ComponentBase”不是泛型类型，要么特性中 Type 参数的数目错误。请移除 x:TypeArguments 特性，因为只在通用类型上允许该特性，或将其值修复为匹配泛型类型“ComponentBase”的 arity。 行 12 位置 63.	ClassIsland	D:\ClassIsland-master\ClassIsland\Controls\Components\WeatherComponent.xaml	12		
错误(活动)	MC6020	标记“ComponentBase”上的 x:TypeArguments='componentSettings:WeatherComponentSettings' 无效。要么“ComponentBase”不是泛型类型，要么特性中 Type 参数的数目错误。请移除 x:TypeArguments 特性，因为只在通用类型上允许该特性，或将其值修复为匹配泛型类型“ComponentBase”的 arity。 行 11 位置 66.	ClassIsland	D:\ClassIsland-master\ClassIsland\Controls\Components\WeatherComponentSettingsControl.xaml	11		
错误(活动)	MC3074	XML 命名空间“clr-namespace:ClassIsland.Core.Converters;assembly=ClassIsland.Core”中不存在标记“SecondsToFormatTimeConverter”。 行 16 位置 10.	ClassIsland	D:\ClassIsland-master\ClassIsland\Controls\LessonControl.xaml	16		
错误(活动)	MC3074	XML 命名空间“clr-namespace:ClassIsland.Core.Converters;assembly=ClassIsland.Core”中不存在标记“WebImageUrlToBitmapImageConverter”。 行 13 位置 10.	ClassIsland	D:\ClassIsland-master\ClassIsland\Controls\MiniInfoProvider\WeatherMiniInfoProviderControl.xaml	13		
错误(活动)	MC3074	XML 命名空间“clr-namespace:ClassIsland.Core.Controls;assembly=ClassIsland.Core”中不存在标记“IconText”。 行 18 位置 14.	ClassIsland	D:\ClassIsland-master\ClassIsland\Controls\NavigationButtonsControl.xaml	18		
错误(活动)	MC3074	XML 命名空间“clr-namespace:ClassIsland.Core.Converters;assembly=ClassIsland.Core”中不存在标记“RippleEffectTranslationConverter”。 行 13 位置 10.	ClassIsland	D:\ClassIsland-master\ClassIsland\Controls\NotificationEffects\RippleEffect.xaml	13		
错误(活动)	MC3074	XML 命名空间“http://classisland.tech/schemas/xaml/core”中不存在标记“IconText”。 行 23 位置 30.	ClassIsland	D:\ClassIsland-master\ClassIsland\Controls\NotificationProviders\AfterSchoolNotificationProviderSettingsControl.xaml	23		
错误(活动)	MC3074	XML 命名空间“clr-namespace:ClassIsland.Core.Converters;assembly=ClassIsland.Core”中不存在标记“TimeSpanToTotalSecondsConverter”。 行 24 位置 10.	ClassIsland	D:\ClassIsland-master\ClassIsland\Controls\NotificationProviders\ClassNotificationProviderControl.xaml	24		
错误(活动)	MC3074	XML 命名空间“clr-namespace:ClassIsland.Core.Converters;assembly=ClassIsland.Core”中不存在标记“IntToStringConverter”。 行 15 位置 10.	ClassIsland	D:\ClassIsland-master\ClassIsland\Controls\NotificationProviders\ClassNotificationProviderSettingsControl.xaml	15		
错误(活动)	MC3074	XML 命名空间“clr-namespace:ClassIsland.Core.Converters;assembly=ClassIsland.Core”中不存在标记“NotificationModeToIntConverter”。 行 16 位置 10.	ClassIsland	D:\ClassIsland-master\ClassIsland\Controls\NotificationProviders\WeatherNotificationProviderSettingsControl.xaml	16		
错误(活动)	MC6020	标记“RuleSettingsControlBase”上的 x:TypeArguments='rules:CurrentSubjectRuleSettings' 无效。要么“RuleSettingsControlBase”不是泛型类型，要么特性中 Type 参数的数目错误。请移除 x:TypeArguments 特性，因为只在通用类型上允许该特性，或将其值修复为匹配泛型类型“RuleSettingsControlBase”的 arity。 行 12 位置 46.	ClassIsland	D:\ClassIsland-master\ClassIsland\Controls\RuleSettingsControls\CurrentSubjectRuleSettingsControl.xaml	12		
错误(活动)	MC6020	标记“RuleSettingsControlBase”上的 x:TypeArguments='rules:TimeStateRuleSettings' 无效。要么“RuleSettingsControlBase”不是泛型类型，要么特性中 Type 参数的数目错误。请移除 x:TypeArguments 特性，因为只在通用类型上允许该特性，或将其值修复为匹配泛型类型“RuleSettingsControlBase”的 arity。 行 13 位置 46.	ClassIsland	D:\ClassIsland-master\ClassIsland\Controls\RuleSettingsControls\TimeStateRuleSettingsControl.xaml	13		
错误(活动)	MC6020	标记“RuleSettingsControlBase”上的 x:TypeArguments='rules:WindowStatusRuleSettings' 无效。要么“RuleSettingsControlBase”不是泛型类型，要么特性中 Type 参数的数目错误。请移除 x:TypeArguments 特性，因为只在通用类型上允许该特性，或将其值修复为匹配泛型类型“RuleSettingsControlBase”的 arity。 行 12 位置 46.	ClassIsland	D:\ClassIsland-master\ClassIsland\Controls\RuleSettingsControls\WindowStatusRuleSettingsControl.xaml	12		
错误(活动)	MC3074	XML 命名空间“clr-namespace:ClassIsland.Core.Controls;assembly=ClassIsland.Core”中不存在标记“IconText”。 行 65 位置 10.	ClassIsland	D:\ClassIsland-master\ClassIsland\Controls\TimeLineListItemControl.xaml	65		
错误(活动)	MC3074	XML 命名空间“http://classisland.tech/schemas/xaml/core”中不存在标记“IconText”。 行 49 位置 22.	ClassIsland	D:\ClassIsland-master\ClassIsland\Controls\WeekOffsetSettingsControl.xaml	49		
错误(活动)	MC3074	XML 命名空间“clr-namespace:ClassIsland.Core.Converters;assembly=ClassIsland.Core”中不存在标记“IndexConverter”。 行 45 位置 10.	ClassIsland	D:\ClassIsland-master\ClassIsland\MainWindow.xaml	45		
错误(活动)	MC3066	类型引用无法找到名为“NumbericTextBox”的公共类型。 行 23 位置 108.	ClassIsland	D:\ClassIsland-master\ClassIsland\Themes\Generic.xaml	23		
错误(活动)	MC3074	XML 命名空间“clr-namespace:ClassIsland.Core.Controls;assembly=ClassIsland.Core”中不存在标记“MyWindow”。 行 1 位置 21.	ClassIsland	D:\ClassIsland-master\ClassIsland\Views\AppLogsWindow.xaml	1		
错误(活动)	MC3074	XML 命名空间“clr-namespace:ClassIsland.Core.Controls;assembly=ClassIsland.Core”中不存在标记“MyWindow”。 行 1 位置 21.	ClassIsland	D:\ClassIsland-master\ClassIsland\Views\ClassChangingWindow.xaml	1		
错误(活动)	MC3074	XML 命名空间“http://classisland.tech/schemas/xaml/core”中不存在标记“MyWindow”。 行 1 位置 14.	ClassIsland	D:\ClassIsland-master\ClassIsland\Views\ClassPlanDetailsWindow.xaml	1		
错误(活动)	MC3074	XML 命名空间“clr-namespace:ClassIsland.Core.Controls;assembly=ClassIsland.Core”中不存在标记“MyWindow”。 行 1 位置 21.	ClassIsland	D:\ClassIsland-master\ClassIsland\Views\CrashWindow.xaml	1		
错误(活动)	MC3074	XML 命名空间“clr-namespace:ClassIsland.Core.Controls;assembly=ClassIsland.Core”中不存在标记“MyWindow”。 行 1 位置 21.	ClassIsland	D:\ClassIsland-master\ClassIsland\Views\ExcelImportWindow.xaml	1		
错误(活动)	MC3074	XML 命名空间“http://classisland.tech/schemas/xaml/core”中不存在标记“MyWindow”。 行 1 位置 14.	ClassIsland	D:\ClassIsland-master\ClassIsland\Views\FeatureDebugWindow.xaml	1		
错误(活动)	MC3074	XML 命名空间“clr-namespace:ClassIsland.Core.Controls;assembly=ClassIsland.Core”中不存在标记“MyWindow”。 行 1 位置 21.	ClassIsland	D:\ClassIsland-master\ClassIsland\Views\ProfileSettingsWindow.xaml	1		
错误(活动)	MC3074	XML 命名空间“clr-namespace:ClassIsland.Core.Abstractions.Controls;assembly=ClassIsland.Core”中不存在标记“SettingsPageBase”。 行 1 位置 28.	ClassIsland	D:\ClassIsland-master\ClassIsland\Views\SettingPages\AboutSettingsPage.xaml	1		
错误(活动)	MC3074	XML 命名空间“clr-namespace:ClassIsland.Core.Abstractions.Controls;assembly=ClassIsland.Core”中不存在标记“SettingsPageBase”。 行 1 位置 28.	ClassIsland	D:\ClassIsland-master\ClassIsland\Views\SettingPages\AppearanceSettingsPage.xaml	1		
错误(活动)	MC3074	XML 命名空间“clr-namespace:ClassIsland.Core.Abstractions.Controls;assembly=ClassIsland.Core”中不存在标记“SettingsPageBase”。 行 1 位置 28.	ClassIsland	D:\ClassIsland-master\ClassIsland\Views\SettingPages\ComponentsSettingsPage.xaml	1		
错误(活动)	MC3074	XML 命名空间“clr-namespace:ClassIsland.Core.Abstractions.Controls;assembly=ClassIsland.Core”中不存在标记“SettingsPageBase”。 行 1 位置 28.	ClassIsland	D:\ClassIsland-master\ClassIsland\Views\SettingPages\DebugBrushesSettingsPage.xaml	1		
错误(活动)	MC3074	XML 命名空间“clr-namespace:ClassIsland.Core.Abstractions.Controls;assembly=ClassIsland.Core”中不存在标记“SettingsPageBase”。 行 1 位置 28.	ClassIsland	D:\ClassIsland-master\ClassIsland\Views\SettingPages\DebugPage.xaml	1		
错误(活动)	MC3074	XML 命名空间“clr-namespace:ClassIsland.Core.Abstractions.Controls;assembly=ClassIsland.Core”中不存在标记“SettingsPageBase”。 行 1 位置 28.	ClassIsland	D:\ClassIsland-master\ClassIsland\Views\SettingPages\GeneralSettingsPage.xaml	1		
错误(活动)	MC3074	XML 命名空间“clr-namespace:ClassIsland.Core.Abstractions.Controls;assembly=ClassIsland.Core”中不存在标记“SettingsPageBase”。 行 1 位置 28.	ClassIsland	D:\ClassIsland-master\ClassIsland\Views\SettingPages\NotificationSettingsPage.xaml	1		
错误(活动)	MC3074	XML 命名空间“http://classisland.tech/schemas/xaml/core”中不存在标记“SettingsPageBase”。 行 1 位置 22.	ClassIsland	D:\ClassIsland-master\ClassIsland\Views\SettingPages\PluginsSettingsPage.xaml	1		
错误(活动)	MC3074	XML 命名空间“clr-namespace:ClassIsland.Core.Abstractions.Controls;assembly=ClassIsland.Core”中不存在标记“SettingsPageBase”。 行 1 位置 28.	ClassIsland	D:\ClassIsland-master\ClassIsland\Views\SettingPages\PrivacySettingsPage.xaml	1		
错误(活动)	MC3074	XML 命名空间“http://classisland.tech/schemas/xaml/core”中不存在标记“SettingsPageBase”。 行 1 位置 22.	ClassIsland	D:\ClassIsland-master\ClassIsland\Views\SettingPages\StorageSettingsPage.xaml	1		
错误(活动)	MC3074	XML 命名空间“clr-namespace:ClassIsland.Core.Abstractions.Controls;assembly=ClassIsland.Core”中不存在标记“SettingsPageBase”。 行 1 位置 28.	ClassIsland	D:\ClassIsland-master\ClassIsland\Views\SettingPages\TestSettingsPage.xaml	1		
错误(活动)	MC3074	XML 命名空间“clr-namespace:ClassIsland.Core.Abstractions.Controls;assembly=ClassIsland.Core”中不存在标记“SettingsPageBase”。 行 1 位置 28.	ClassIsland	D:\ClassIsland-master\ClassIsland\Views\SettingPages\UpdatesSettingsPage.xaml	1		
错误(活动)	MC3074	XML 命名空间“clr-namespace:ClassIsland.Core.Abstractions.Controls;assembly=ClassIsland.Core”中不存在标记“SettingsPageBase”。 行 1 位置 28.	ClassIsland	D:\ClassIsland-master\ClassIsland\Views\SettingPages\WeatherSettingsPage.xaml	1		
错误(活动)	MC3074	XML 命名空间“clr-namespace:ClassIsland.Core.Abstractions.Controls;assembly=ClassIsland.Core”中不存在标记“SettingsPageBase”。 行 1 位置 28.	ClassIsland	D:\ClassIsland-master\ClassIsland\Views\SettingPages\WindowSettingsPage.xaml	1		
错误(活动)	MC3074	XML 命名空间“clr-namespace:ClassIsland.Core.Controls;assembly=ClassIsland.Core”中不存在标记“MyWindow”。 行 1 位置 20.	ClassIsland	D:\ClassIsland-master\ClassIsland\Views\SettingsWindowNew.xaml	1		
错误(活动)	MC3074	XML 命名空间“clr-namespace:Walterlv.Windows;assembly=ClassIsland.Core”中不存在标记“AsyncBox”。 行 27 位置 6.	ClassIsland	D:\ClassIsland-master\ClassIsland\Views\SplashWindow.xaml	27		
错误(活动)	MC3074	XML 命名空间“http://classisland.tech/schemas/xaml/core”中不存在标记“MyWindow”。 行 1 位置 14.	ClassIsland	D:\ClassIsland-master\ClassIsland\Views\UpdateProgressWindow.xaml	1		
错误(活动)	MC3074	XML 命名空间“clr-namespace:ClassIsland.Core.Controls;assembly=ClassIsland.Core”中不存在标记“MyWindow”。 行 1 位置 21.	ClassIsland	D:\ClassIsland-master\ClassIsland\Views\WallpaperPreviewWindow.xaml	1		
错误(活动)	MC3074	XML 命名空间“clr-namespace:ClassIsland.Core.Controls;assembly=ClassIsland.Core”中不存在标记“MyWindow”。 行 1 位置 21.	ClassIsland	D:\ClassIsland-master\ClassIsland\Views\WelcomeWindow.xaml	1		
错误(活动)	MC3074	XML 命名空间“clr-namespace:ClassIsland.Core.Controls;assembly=ClassIsland.Core”中不存在标记“MyWindow”。 行 1 位置 21.	ClassIsland	D:\ClassIsland-master\ClassIsland\Views\WindowsPicker.xaml	1		
错误(活动)	MSB4057	该项目中不存在目标“GitVersion”。	ClassIsland.PluginSdk	D:\ClassIsland-master\Global.props	13
